js/components/controlBar.jsx

```diff
@@ -545,6 +546,7 @@ class ControlBar extends React.Component {
             icon={volumeIcon}
             tooltip={mutedInUi ? CONSTANTS.SKIN_TEXT.UNMUTE : CONSTANTS.SKIN_TEXT.MUTE}
             onClick={this.handleVolumeIconClick}
+            role="radio"
```

Comment:
Does it work if you try to mute and then unmute using the keyboard only?

Reply:
I am interested if it works with other readers, but I guess it does.

Reply:
Yes it will work. Also it was working before but the mute button was losing only visual focus overlay. The Play/Pause button was loosing visual and actual focus.